aws-lc-rs/src/cipher.rs

```diff
@@ -632,7 +706,8 @@ fn decrypt<'in_out>(
     let block_len = algorithm.block_len();
 
     match mode {
-        OperatingMode::CTR => {}
+        // TODO: Hopefully support CFB1, and CFB8
+        OperatingMode::CTR | OperatingMode::CFB128 => {}
         _ => {
             if (in_out.len() % block_len) != 0 {
                 return Err(Unspecified);
```

Comment:
NP: I think it would be better to make CBC mode explicit, and the default case the no-op (or make all modes explicit?):
```
    match mode {
        OperatingMode::CBC => {
            if (in_out.len() % block_len) != 0 {
                return Err(Unspecified);
            }
        }
        _ => {}
    }
```